src/ert/ensemble_evaluator/builder/_legacy.py

```diff
@@ -13,10 +13,12 @@
     Optional,
     Sequence,
     Tuple,
+    Union,
 )
 
 from cloudevents.http.event import CloudEvent
 
+import ert.experiment_server as es
```

Comment:
We are closing in to a circular dependency here I think, perhaps we can extract the necessary functionality from experiment server (creating messages etc) and make it available for all consumers.

Reply:
The protobuf statemachine has been moved to `ert.protobuf` module.